tests/integration/integration/login/login-en.spec.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from "testcafe";
```

Comment:
why have we kept the folder structure as `integration/integration`?

Reply:
I had copied it from cypress where it came with it's own `integration` folder having test cases. We can rename the folder that keeps the test cases to something else if you prefer. Like `integrations/test-cases`?

Reply:
Ah, can't we simply use `tests/integration/login.spec`?